neo4j/neo4j.go

```diff
@@ -56,7 +56,7 @@ func New(dbAddr string, size, timeout, retries int) (n *Neo4j, err error) {
 func (n *Neo4j) checkAttempts(err error, instanceID string, attempt int) error {
 	ctx := context.Background()
 	if !isTransientError(err) {
-		log.Event(ctx, "received an error from neo4j that cannot be retried", log.ERROR,
+		log.Error(ctx, "received an error from neo4j that cannot be retried", err,
 			log.Data{"instance_id": instanceID, "error": err})
```

Comment:
I think formatting wise it's better to close the brackets on the same column they're opened.. also arguably when lines are too long it's better to align all the arguments vertically rather than go to the next line at an arbitrary point

Reply:
Or maybe where log.Data opens
```
log.Error(ctx, "received an error from neo4j that cannot be retried", err, log.Data{
	"instance_id": instanceID,
	"error": err,
})
```

Reply:
That is the style I'm used to writing - just FYI (as their are going to be a bunch of these PRs, and you've been good enough to review a few so far [thanks!]) for the logging statement refactor we're trying to do minimal changes to the original code, just swap the logging calls out. I can't see any issues with re-styling this bit though. Will change.

Reply:
resolving this conversation as its stale - see comment in new code for further discussion